packages/atlas/src/views/global/YppDashboard/YppDashboard.tsx

```diff
@@ -0,0 +1,174 @@
+import BN from 'bn.js'
+import { FC } from 'react'
+
+import { Banner } from '@/components/Banner'
+import { Information } from '@/components/Information'
+import { LimitedWidthContainer } from '@/components/LimitedWidthContainer'
+import { Text } from '@/components/Text'
+import { WidgetTile } from '@/components/WidgetTile'
+import {
+  SvgActionInfo,
+  SvgActionNewTab,
+  SvgActionSpeech,
+  SvgActionTokensStack,
+  SvgTierIcon1,
+  SvgTierIcon2,
+  SvgTierIcon3,
+} from '@/components/_icons'
+import { BenefitCard } from '@/components/_ypp/BenefitCard'
+import { useMediaMatch } from '@/hooks/useMediaMatch'
+import {
+  Header,
+  RewardsWrapper,
+  StyledSvgAlertsInformative24,
+  TierCount,
+  TierDescription,
+  TierWrapper,
+  WidgetsWrapper,
+} from '@/views/global/YppDashboard/YppDashboard.styles'
+
+const REWARDS = [
```

Comment:
Could we maybe move this and `TIERS` to a separate file to improve component's readability? Can be called `YppDashboard.config.ts`